testvectorgen/contract_data/generate_json_test.go

```diff
@@ -378,12 +375,27 @@ func generateMTPData(t *testing.T, desc string, gistData []*gistData, nextState
 	subjectID := user.ID
 	nonceSubject := big.NewInt(0)
 
-	claim := utils.DefaultUserClaim(t, subjectID)
+	mz, claim := utils.DefaultJSONUserClaim(t, subjectID)
```

Comment:
Is comparable with this https://github.com/iden3/go-iden3-auth/pull/28/files#diff-085dc8c040d6958576b201be11be5fe556690be79dea15a0e0a510d85d414770R101?